Makefile

```diff
@@ -93,12 +93,8 @@ compiledmessages:
 	cd src/oscar; django-admin.py compilemessages
 
 css:
-	# Compile CSS files from LESS
-	lessc --source-map --source-map-less-inline src/oscar/static/oscar/less/styles.less src/oscar/static/oscar/css/styles.css
-	lessc --source-map --source-map-less-inline src/oscar/static/oscar/less/dashboard.less src/oscar/static/oscar/css/dashboard.css
-	# Compile CSS for demo site
-	lessc --source-map --source-map-less-inline sites/demo/static/demo/less/styles.less sites/demo/static/demo/css/styles.css
-	lessc --source-map --source-map-less-inline sites/demo/static/demo/less/responsive.less sites/demo/static/demo/css/responsive.css
+	npm install
+	npm run compile
```

Comment:
I know we used to call it `compile`, mostly out of ignorance. But I think a name like `preprocess-css` is much more appropriate.